### DISCUSSION TITLE
Getting error while building the Github code

### DISCUSSION THREAD

**Discussion by ChandruforD365** [community/community#76209] [Programming Help]
### Body

I have downloaded the entire code from the following path https://github.com/microsoft/fsi-experiences/tree/main/.

While build the code facing the below error
<img width="638" alt="Github-Error" src="https://github.com/community/community/assets/87597482/f8342ac5-8069-4fe0-ada3-c31148db3e4d">

Error 1:

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3073	The command "G:\fsi-experiences-main\packages\microsoft.crmsdk.coretools\9.1.0.155\content\bin\coretools\SolutionPackager.exe /action:pack /packagetype:both /map:packageMap.xml /folder:G:\fsi-experiences-main\Modules\UCP-Wealth-Management\FSIWealthManagement /zipfile:G:\fsi-experiences-main\\target\Debug\AnyCPU\UCP-Wealth-Management\FSIWealthManagement\Debug\FSIWealthManagement.zip /errorlevel:Verbose /l:G:\fsi-experiences-main\\target\Debug\AnyCPU\UCP-Wealth-Management\FSIWealthManagement\Debug\FSIWealthManagement_packager.log" exited with code 1.	FSIWealthManagement	G:\fsi-experiences-main\build\include\Localization.targets	43	

**Error 2:**

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3073	The command "G:\fsi-experiences-main\packages\microsoft.crmsdk.coretools\9.1.0.155\content\bin\coretools\SolutionPackager.exe /n /action:extract /packagetype:both /folder:G:\fsi-experiences-main\\target\Debug\AnyCPU\UCP-Retail-Banking\FSIRetailBankingCoreComponents\Debug\LocExt /zipfile:G:\fsi-experiences-main\\target\Debug\AnyCPU\UCP-Retail-Banking\FSIRetailBankingCoreComponents\Debug\FSIRetailBankingCoreComponents.zip /errorLevel:verbose /src /loc" exited with code 1.	FSIRetailBankingCoreComponents	G:\fsi-experiences-main\build\include\Localization.targets	47	

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ZhengLinLei** [comment]
Hi @ChandruforD365,

Could you please check the logs for those exited programs? `SolutionPackager.exe`

**Comment by ChandruforD365** [comment]
Hi Zheng Lin Lei,

PFA.

Regards,
Chandrasekaran R

**Reply by ZhengLinLei** [reply]
You are attaching a screenshot of exit 1 in execution of those programs. But the details are in the log files, without the log files I cannot find the error caused this exit.

Thanks.
